core/src/main/resources/reference.conf

```diff
@@ -206,183 +69,219 @@ cassandra-journal {
   protocol-version = ""
 
-
-  # Enable Dropwizard Metrics for Cluster.
-  # See: https://docs.datastax.com/en/developer/java-driver/3.5/manual/metrics/
-  metrics-enabled = on
-
-  # Enable JMX reporter for Dropwizard Metrics.
-  # See: https://docs.datastax.com/en/developer/java-driver/3.5/manual/metrics/
-  jmx-reporting-enabled = on
-
   # If there is an unexpected error in the Cassandra Journal the journal shuts down
   # to prevent any data corruption. Set to true to also run coordinated shutdown so
   # that the application shuts down. Useful in environments where applicatons are
   # automatically shutdown or if the Cassandra Journal not working means the application
   # won't function.
   coordinated-shutdown-on-error = off
-}
-
-# This configures the default settings for all Cassandra Snapshot plugin
-# instances in the system. If you are using just one configuration for
-# all persistent actors then you should point your akka.persistence.snapshot-store.plugin
-# setting to this section.
-#
-# Otherwise you need to create differently named sections containing
-# only those settings that shall be different from the defaults
-# configured here, importing the defaults like so:
-#
-#   my-cassandra-snapshot-store = ${cassandra-snapshot-store}
-#   my-cassandra-snapshot-store {
-#     <settings...>
-#   }
-cassandra-snapshot-store {
-
-  # FQCN of the cassandra snapshot store plugin
-  class = "akka.persistence.cassandra.snapshot.CassandraSnapshotStore"
-
-  session-provider = "akka.cassandra.session.DefaultSessionProvider"
-
-  session-name = ""
-
-  write-profile = "cassandra-journal"
-  read-profile = "cassandra-journal"
 
-  # Name of the keyspace to be created/used by the snapshot store
-  keyspace = "akka_snapshot"
-
-  # Parameter indicating whether the snapshot keyspace should be auto created
-  # Not all Cassandra settings are configurable when using autocreate and for
-  # full control of the keyspace and table definitions you should create them
-  # manually (with a script).
-  keyspace-autocreate = true
-
-  # Parameter indicating whether the snapshot tables should be auto created
-  # Not all Cassandra settings are configurable when using autocreate and for
-  # full control of the keyspace and table definitions you should create them
-  # manually (with a script).
-  tables-autocreate = true
-
-  # Name of the table to be created/used by the snapshot store.
-  # If the table doesn't exist it is automatically created.
-  table = "snapshots"
-
-  # Compaction strategy for the snapshot table
-  # Please refer to the tests for example configurations.
-  # Refer to http://docs.datastax.com/en/cql/3.1/cql/cql_reference/compactSubprop.html
-  # for more information regarding the properties.
-  table-compaction-strategy {
-    class = "SizeTieredCompactionStrategy"
-  }
-
-  # Name of the table to be created/used for journal config.
-  # If the table doesn't exist it is automatically created.
-  config-table = "config"
+  write {
```

Comment:
something feels off about cassandra-journal.write

What about cassandra-plug.journal, cassandra-plugin.query, casandra-plugin.snapshot?

Reply:
good suggestion, I like it